### ISSUE TITLE
Silence option from Homekit for Protect

### ISSUE THREAD

**Issue by techoin** [chrisjshull/homebridge-nest#133]
Is it possible to have silence button available in Homekit when nest protect fire alarm is triggered?

Currently Homekit shows triggered notification in case smoke/co alert but silence option is only available in Nest app. 

Also Homekit provides notification in case of Smoke Alert but does not notify once Smoke is cleared.

**Comment by adriancable** [comment]
@techoin - I like the silence switch idea. I'll work on it.

Regarding notifying once smoke is cleared, I don't have any control over HomeKit's notifications. I think that is a HomeKit "feature" unfortunately, that you don't get a notification once smoke is cleared. Presumably once the smoke has cleared, the "triggered" status goes to off.

**Comment by adriancable** [comment]
@techoin - I have just looked into this, and it isn't possible to silence Nest Protects remotely in the way I thought. When you use the 'silence' option in the Nest app, it communicates directly with the Nest Protect via Bluetooth (rather than though the API). So, there is no way to duplicate this capability in homebridge-nest unfortunately.

Thanks for the suggestion though - I wish it was possible! Closing for now.
